lib/jellyfish_web/controllers/peer_controller.ex

```diff
@@ -67,7 +67,7 @@ defmodule JellyfishWeb.PeerController do
     # pid and adding a new peer to it
     # in such a case, the controller will fail
     # and Phoenix will return 500
-    with peer_options <- Map.get(params, "options", %{}),
```

Comment:
Maybe I missed something but this change just adds one additional `if` statement to the `room.ex` file.

Reply:
That's for consistency with changes from component_controller.ex in https://github.com/jellyfish-dev/jellyfish/pull/82

Reply:
I think we should go back to `Map.get(params, "options", %{})`, both when fetching peer options as well as component options.
My previous change was motivated by the fact, that the `options` could be `nil`, so a default value of `nil` made sense.
Now, neither peer nor component options are nullable, so the default value can be an empty map.

Reply:
Changed